### PR TITLE
Redesign: api integration

### PR DESCRIPTION
**Pull request by bbo-git**
- some localization
- getting data on personal info screen 
- personal info screen add viewmodel

### CODE REVIEW

File: Sources/EduIDExpansion/Empty.md [hunk 1]
```diff
@@ -0,0 +1,444 @@
+import I18n from "i18n-js";
```
Comment (johankool) [inline Sources/EduIDExpansion/Empty.md:1]: This file is neither empty, nor markdown. 😄

File: Sources/EduIDExpansion/Flows/PersonalInfo/PersonalInfoViewController.swift [hunk 2]
```diff
@@ -9,13 +10,54 @@ class PersonalInfoViewController: UIViewController, ScreenWithScreenType {
...
+                views.forEach { view in
+                    self?.stack.removeArrangedSubview(view)
+                }
+            }
+        }
+        
+        viewModel.dataFetchErrorClosure = { [weak self] error in
+            let alert = UIAlertController(title: "Error", message: error.localizedDescription, preferredStyle: .alert)
+            self?.present(alert, animated: true)
+            DispatchQueue.main.asyncAfter(deadline: .now() + 3) {
```
Comment (johankool) [inline Sources/EduIDExpansion/Flows/PersonalInfo/PersonalInfoViewController.swift:45]: Auto dismissal of an error seems like unexpected behavior to me. Was this a requirement?

File: Sources/EduIDExpansion/Flows/PersonalInfo/PersonalInfoViewController.swift [hunk 3]
```diff
@@ -34,60 +76,84 @@ class PersonalInfoViewController: UIViewController, ScreenWithScreenType {
         scrollView.edges(to: view)
         
         // - posterLabel
-        let posterLabel = UILabel.posterTextLabelBicolor(text: "Your personal info", size: 24, primary: "Your personal info")
+        let posterLabel = UILabel.posterTextLabelBicolor(text: NSLocalizedString(LocalizedKey.Profile.title, bundle: .module, comment: ""), size: 24, primary: NSLocalizedString(LocalizedKey.Profile.title, bundle: .module, comment: ""))
```
Comment (johankool) [inline Sources/EduIDExpansion/Flows/PersonalInfo/PersonalInfoViewController.swift:79]: Perhaps an extension on LocalizedKey would make this more readable? 

```swift
extension LocalizedKey {
  var localization: String {
    NSLocalizedString(self, bundle: .module, comment: "")
  }
```

so you can just use `LocalizedKey.Profile.title.localization`

File: Sources/EduIDExpansion/Flows/PersonalInfo/ViewModels/PersonalInfoViewModel.swift [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+            let nameProvidedBy = firstLinkedAccount.schacHomeOrganization ?? ""
+            let model = PersonalInfoDataCallbackModel(userResponse: userResponse, name: name, nameProvidedBy: nameProvidedBy, isNameProvidedByInstitution: true)
+            
+            dataAvailableClosure?(model)
+        }
+    }
+    
+    func removeLinkedAccount(linkedAccount: LinkedAccount) {
+        Task {
+            do {
```
Comment (johankool) [inline Sources/EduIDExpansion/Flows/PersonalInfo/ViewModels/PersonalInfoViewModel.swift:55]: Unneeded do (I think).

File: Sources/EduIDExpansion/LocalizedKey.swift [hunk 5]
```diff
@@ -0,0 +1,641 @@
+//
+//  File.swift
```
Comment (johankool) [inline Sources/EduIDExpansion/LocalizedKey.swift:2]: Wrong name

**Review by johankool — APPROVED** [review]
Looks good. One suggestion to make it perhaps a bit more readable. And with the note that a few strings remains unlocalized.

File: Sources/EduIDExpansion/Flows/PersonalInfo/PersonalInfoViewController.swift (hunk 3 above)
Comment (bbo-git) [reply Sources/EduIDExpansion/Flows/PersonalInfo/PersonalInfoViewController.swift:79]: nice!

File: Sources/EduIDExpansion/Flows/PersonalInfo/ViewModels/PersonalInfoViewModel.swift (hunk 4 above)
Comment (bbo-git) [reply Sources/EduIDExpansion/Flows/PersonalInfo/ViewModels/PersonalInfoViewModel.swift:55]: what about handling the error, this is a case of async method, I was under the impression that the error is embedded in a do / catch pattern

File: Sources/EduIDExpansion/Flows/PersonalInfo/PersonalInfoViewController.swift (hunk 2 above)
Comment (bbo-git) [reply Sources/EduIDExpansion/Flows/PersonalInfo/PersonalInfoViewController.swift:45]: good remark, I will add an ok-button

**Pull request merged**